src/lib/util.js

```diff
@@ -158,7 +219,20 @@ export function get(url, ...rest) {
 
     return fetch(url, { headers })
         .then(checkStatus)
-        .then(parser);
+        .then(parser)
+        .catch((e) => {
+            // Make sure we are making a dl call
+            if (isNonDefaultDownloadHost(url)) {
```

Comment:
This checks to make sure that this URL is a dl request in addition to it not being the default. This means we don't have to also check if the session storage has the flag.